features/fixtures/maze_runner/Assets/Scripts/Scenarios/Fallback/Breadcrumbs/DisableBreadcrumbs.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using BugsnagUnity;
+using BugsnagUnity.Payload;
+using System;
+
+public class DisableBreadcrumbs : Scenario
+{
+    public override void PrepareConfig(string apiKey, string host)
+    {
+        base.PrepareConfig(apiKey, host);
+        Configuration.EnabledBreadcrumbTypes = new BreadcrumbType[0];
+    }
+
+    public override void Run()
+    {
+        base.Run();
```

Comment:
I think you can remove all these calls to `base.Run()`, they just add noise.

Reply:
done 👍